server/container_create.go

```diff
@@ -359,6 +359,19 @@ func (s *Server) CreateContainer(ctx context.Context, req *types.CreateContainer
 		}, nil
 	}
 
+	systemCtx, err := s.contextForNamespace(req.SandboxConfig.Metadata.Namespace)
+	if err != nil {
+		return nil, fmt.Errorf("get context for namespace: %w", err)
+	}
+
+	if err := s.StorageImageServer().IsRunningImageAllowed(ctx, &systemCtx, req.Config.Image.UserSpecifiedImage); err != nil {
+		return nil, err
+	}
+	log.Debugf(ctx,
+		"Is allowed to run config image %s (user requested image: %s, policy path: %s)",
+		req.Config.Image.Image, req.Config.Image.UserSpecifiedImage, systemCtx.SignaturePolicyPath,
```

Comment:
`%q` for policy path?

But really it would make a bit more sense to me to have this kind of logging inside `IsRunningImageAllowed`; that function is more authoritative as to which inputs / environment values are relevant for the policy check, and it would mean the logging is consistent (right now ImagePull is not logging this).

Reply:
Moved that log into `IsRunningImageAllowed`.